azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -116,6 +121,19 @@ public void cleanUpStaleFlows() {
         Entry::getKey).forEach(this::cleanUpStaleFlows);
   }
 
+  /**
+   * Try to clean the stale containers that are older than maximum azkaban flow running time
+   */
+  public void cleanUpStaleContainers() {
+    try {
+      this.containerizedImpl.deleteAgedContainers(validityMap.get(Status.RUNNING).getFirst());
+    } catch (ExecutorManagerException eme) {
+      logger.error("ExecutorManagerException occurred while deleting stale pods and services." + eme);
+    } catch (RuntimeException re) {
+      logger.error("Unexpected RuntimeException while finalizing flow during clean up." + re);
+    }
```

Comment:
Do we need to treat these exceptions differently? Why not
`catch (ExecutorManagerException | RuntimeException e ) {..}`
or just 
`catch (Exception e) {..}`

Reply:
This makes sense. Made changes to catch general Exception.